cpp/cmake_modules/Findlibrados.cmake

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
It's unfortunate there is no librados2 in conda-forge.  I tried to make it work with a hybrid build but my system librados2 linked with system libcrypto while parquet linked with conda's libssl and the mismatch caused errors.  I ended up making a cmake build that used all system libraries and that worked.